### PR TITLE
Issue #SC-545,SC-682 feat: Configuration for custodian org and tnc

### PR DESCRIPTION
**Pull request by bvinayakumar**

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @bvinayakumar,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://394.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://394.qa.docs.sunbird.org/master/developer-docs/configuring_sunbird/configuring_custodian_org/ <br /> - http://394.qa.docs.sunbird.org/master/developer-docs/configuring_sunbird/configuring_terms_and_conditions/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @bvinayakumar,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://394.qa.docs.sunbird.org/master/ .<br/> **Note:** Only _master_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://394.qa.docs.sunbird.org/master/developer-docs/configuring_sunbird/configuring_custodian_org/ <br /> - http://394.qa.docs.sunbird.org/master/developer-docs/configuring_sunbird/configuring_terms_and_conditions/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/configuring_sunbird/configuring_custodian_org.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: Configuring Custodian Organisation
```
Comment (seventunes) [inline developer-docs/configuring_sunbird/configuring_custodian_org.md]: test

**Comment by Sunitashelar** [comment]
closing the PR without merging it

**Pull request closed without merging**